### PR TITLE
fix: [CCM-7054]: show error in Connection Test step of CE AWS Connector

### PR DESCRIPTION
**Pull request by rajarshee-harness**
##### Jira Links:

https://harness.atlassian.net/browse/CCM-7054

##### Screenshots:

Screenshot 2022-03-28 at 10 47 46 AM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by rajarshee-harness** [comment]
trigger rebase

File: src/modules/35-connectors/components/CreateConnector/CENGAwsConnector/steps/TestConnection.tsx [hunk 1]
```diff
@@ -92,7 +84,7 @@ const TestConnection: React.FC<TestConnectionProps> = props => {
 
   return (
     <Layout.Vertical className={css.stepContainer}>
-      <ModalErrorHandler bind={setModalErrorHandler} />
+      {error && <TestConnectionErrorHandler errorMessage={error} />}
```
Comment (jenil-harness) [inline src/modules/35-connectors/components/CreateConnector/CENGAwsConnector/steps/TestConnection.tsx:87]: What was the issue with this?

Comment (rajarshee-harness) [reply src/modules/35-connectors/components/CreateConnector/CENGAwsConnector/steps/TestConnection.tsx:87]: `modalErrorHandler` was undefined when `modalErrorHandler?.showDanger(e.data?.message)` was being called in `useEffect`, so it was not showing the error.

**Comment by rajarshee-harness** [comment]
retrigger jest

**Comment by rajarshee-harness** [comment]
trigger cypress

**Comment by rajarshee-harness** [comment]
retrigger jest

**Comment by rajarshee-harness** [comment]
retrigger coverage

**Review by jenil-harness — APPROVED** [review]

File: src/modules/35-connectors/components/CreateConnector/CENGAwsConnector/steps/TestConnection.tsx [hunk 2]
```diff
@@ -78,17 +78,20 @@ const TestConnection: React.FC<TestConnectionProps> = props => {
         setCurrentStatus(Status.DONE)
         setCurrentStep(3)
       } else {
-        throw new Error('connectors.ceAws.testConnection.error')
+        throw new Error(result.data?.errorSummary)
       }
-    } catch (e) {
-      modalErrorHandler?.showDanger(e.data?.message)
+    } catch (e: any) {
```
[the remark this answers is not in the document]
Comment (rajarshee-harness) [reply src/modules/35-connectors/components/CreateConnector/CENGAwsConnector/steps/TestConnection.tsx:83]: It gives an error as the `catch` clause variable can only be of type any or unknown.

**Review by avinashmadhwani02 — APPROVED** [review]

**Pull request merged**